uniffi_aries_vcx/demo/app/src/main/java/org/hyperledger/ariesvcx/HomeScreen.kt

```diff
@@ -0,0 +1,69 @@
+package org.hyperledger.ariesvcx
+
+import android.os.Handler
+import android.os.Looper
+import android.widget.Toast
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.Button
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.navigation.NavHostController
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.withContext
+
+@Composable
+fun HomeScreen(navController: NavHostController, setProfileHolder: (ProfileHolder) -> Unit, profileHolder: ProfileHolder?, connection: Connection?) {
+
+    val scope = rememberCoroutineScope()
+    val context = LocalContext.current
+
+    var walletConfigState by remember {
+        mutableStateOf(WalletConfig(
+            walletName = "test_create_wallet_add_uuid_here",
+            walletKey = "8dvfYSt5d1taSd6yJdpjq4emkwsPDDLYxkNFysFD2cZY",
+            walletKeyDerivation = "RAW",
+            walletType = null,
+            storageConfig = null,
+            storageCredentials = null,
+            rekey = null,
+            rekeyDerivationMethod = null
+        ))
+    }
```

Comment:
this probably does not need to be a mutableState. i think just `val walletConfig = WalletConfig` is fine (since this variable will not change)